pkg/util/typeutil/schema.go

```diff
@@ -287,6 +287,15 @@ func IsIntegerType(dataType schemapb.DataType) bool {
 	}
 }
 
+func IsJSONType(dataType schemapb.DataType) bool {
+	switch dataType {
+	case schemapb.DataType_Array, schemapb.DataType_JSON:
```

Comment:
Array types are also considered as JSON?

Reply:
Now json and array should be handled the same.